Atomic/backends/_docker.py

```diff
@@ -662,13 +698,15 @@ def _running(self, con_obj, args, atomic):
     def _start(self, con_obj, args, atomic):
         if con_obj.interactive:
             if args.command:
-                util.check_call(
-                    [atomic.docker_binary(), "start", con_obj.name],
-                    stderr=DEVNULL)
+                util.check_call([atomic.docker_binary(), "start", con_obj.name], stderr=DEVNULL)
                 container_command = args.command if isinstance(args.command, list) else args.command.split()
-                return util.check_call(
-                    [atomic.docker_binary(), "exec", "-t", "-i", con_obj.name] +
-                    container_command)
+                try:
+                    return util.check_call([atomic.docker_binary(), "exec", "-t", "-i", con_obj.name] + container_command)
+                except CalledProcessError as e:
+                    if args.debug:
+                        util.write_out(str(e))
+                    raise AtomicError("Unable to execute your command.  Try re-running your command with the --replace"
```

Comment:
I think you should add some more info here.  This error should only happen if the exec failed because the container is not running.
"Failed to execute the command inside the existing container.  In some situations this can happen because the entry point command of the container only runs for a short time.  You might want to replace the container by executing your command with --replace.  Note any updates to the existing container will be lost."